xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -784,6 +789,11 @@ void CMusicInfoTag::SetSongVideoURL(std::string_view songVideoURL)
   m_songVideoURL = songVideoURL;
 }
 
+void CMusicInfoTag::SetChapterMarks(ChapterMarks& chapters)
```

Comment:
I agree with copilots comment:
```suggestion
void CMusicInfoTag::SetChapterMarks(const ChapterMarks& chapters)
```

Reply:
Yeah, missed that const.  It's now done and I took on board your suggestion of `std::chrono::milliseconds` so I have moved to using that.  Thanks for the suggestion :)